tests/yam/agama/agama.pm

```diff
@@ -26,11 +23,9 @@ sub run {
     assert_script_run("/usr/share/agama/system-tests/" . $test . ".cjs", timeout => 1200);
     script_run("dmesg --console-on");
 
-    select_console 'displaymanager';
-    save_screenshot();
-
-    assert_screen('agama-install-finished', 10);
-    assert_and_click('reboot');
+    my $reboot_page = $testapi::distri->get_reboot_page();
```

Comment:
this is initialization, should go under line 20.

Reply:
Solved 257734447a4af251e62c1d1826ecd73a7b8f262a